.github/workflows/ci.yml

```
@@ -9,6 +9,10 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ 20, 24 ]
```

Please remove extra spaces

```suggestion
        node: [20, 24]
```

---

> Please remove extra spaces

fixed